tests/test_level/test_level_06.py

```diff
@@ -535,15 +535,14 @@ def test_print_subtraction_without_text(self):
 
     def test_print_subtraction_with_text(self):
         code = "print 'And the winner is ' 5 - 5"
-        expected_1 = "print(f'And the winner is {int(5) - int(5)}')"
-        expected_2 = "print(f'And the winner is 0')"
+        expected = "print(f'And the winner is {int(5) - int(5)}')"
         output = 'And the winner is 0'
 
-        for level in range(self.level, 12):
-            print(f"passed for level {level}")
-            result = hedy.transpile(code, level, 'en')
-            self.assertIn(result.code, [expected_1, expected_2])
-            self.assertEqual(output, HedyTester.run_code(result))
+        self.multi_level_tester(
+            max_level=11,
```

Comment:
I think we should also test this for level 12 and up.

Reply:
Good catch!! 

The reason that this runs until 11 only is that the output is different at level 12 (`'''` instead of `'`) This is true for a few more tests and @boryanagoncharenko is still working on a better solution but let's indeed not make it worse by skipping test cases in the name of "we will improve this later" haha.